src/common/grpc-expr/src/alter.rs

```diff
@@ -92,7 +92,10 @@ pub fn alter_expr_to_request(expr: AlterExpr) -> Result<AlterTableRequest> {
     }
 }
 
-pub fn create_table_schema(expr: &CreateTableExpr) -> Result<RawSchema> {
+pub fn create_table_schema_options(
```

Comment:
nitpick: Could we rename it to create_table_schema?
```suggestion
pub fn create_table_schema(
```

Reply:
Yessir🫡